routes/road-properties.js

```diff
@@ -18,29 +18,77 @@ function getHandler (req, res) {
     )));
 }
 
-function deleteHandler (req, res) {
-  // This method will have issues if commas are used in road IDs
-  const idsToDelete = req.query.ids && req.query.ids.split(',');
-  if (!idsToDelete) {
-    return res(Boom.badRequest('Must provide a querystring list of road `ids` to delete'));
-  }
-  if (new Set(idsToDelete).size < idsToDelete.length) {
-    return res(Boom.badRequest('Cannot have duplicate road `ids`'));
-  }
-
+function getIdsHandler(req, res) {
   knex('road_properties')
-    .whereIn('id', idsToDelete)
     .select('id')
-    .then(idsInDB => {
-      if (idsInDB.length < idsToDelete.length) {
-        return res(Boom.badRequest(`Some requested road IDs not present in database: ${JSON.stringify(_.difference(idsToDelete, idsInDB))}`));
-      } else {
-        knex('road_properties')
-          .whereIn('id', idsToDelete)
-          .delete()
-          .then(() => res(idsToDelete));
-      }
-    });
+    .map(entry => entry.id)
+  .then(res);
+}
+
+function createHandler(req, res) {
+  knex('road_properties')
+    .insert({
+      id: req.params.road_id,
+      properties: '{}'
+    })
+  .then(function(response) {
+    return res({ id: req.params.road_id }).type('application/json');
+  })
+  .catch(function(err) {
+    if (err.constraint === 'road_properties_pkey') {
+      return res(Boom.conflict());
+    }
+
+    return res(Boom.badImplementation());
+  });
+}
+
+function moveHandler(req, res) {
+  knex('road_properties')
```

Comment:
Won't this fail if `road_properties.id` is used in `point_properties.road_id` or `field_data_geometries.road_id`? I think you have to do something like:

- set related pp.road_id and fdg.road_id to null (save the `id`s of those records)
- update id in `road_properties`
- update related pp.road_id and fdg.road_id to the new `road_properties.road_id`